### PR TITLE
#23971 IPUtils Test failure from incorrect state of IPUtils.disabledIpPrivateSubnet

### PR DESCRIPTION
**Pull request by spbolton**
Fix for #23971

### CODE REVIEW

**Review by wezell — APPROVED** [review]
Looks good.

**Review by wezell — APPROVED** [review]
This looks good.

File: .github/actions/build-core/src/core-builder.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ const COMMANDS: Commands = {
   gradle: [
     {
       cmd: gradleCmd,
-      args: ['createDistPrep'],
+      args: ['createDistPrep', '--stacktrace'],
```
Comment (victoralfaro-dotcms) [inline .github/actions/build-core/src/core-builder.ts:24]: Maybe the `stacktrace` mode should be some parameter we can send from the actual GH workflow file.

File: .github/actions/run-integration-tests/dist/index.js [hunk 2]
```diff
@@ -86,7 +86,7 @@ exports.COMMANDS = {
     gradle: [
         {
             cmd: './gradlew',
-            args: ['integrationTest', `-PdatabaseType=${dbType}`],
+            args: ['integrationTest', `-PdatabaseType=${dbType}`, '--stacktrace'],
```
Comment (victoralfaro-dotcms) [inline .github/actions/run-integration-tests/dist/index.js:89]: Same as comment above.

File: .github/actions/run-integration-tests/src/integration.ts [hunk 3]
```diff
@@ -84,7 +84,7 @@ export const COMMANDS: Commands = {
   gradle: [
     {
       cmd: './gradlew',
-      args: ['integrationTest', `-PdatabaseType=${dbType}`],
+      args: ['integrationTest', `-PdatabaseType=${dbType}`, '--stacktrace'],
```
Comment (victoralfaro-dotcms) [inline .github/actions/run-integration-tests/src/integration.ts:87]: Same as comment above.

File: .github/actions/run-unit-tests/dist/index.js [hunk 4]
```diff
@@ -127,7 +127,7 @@ const reportDir = `${dotCmsRoot}/build/reports/tests/test`;
 exports.COMMANDS = {
     gradle: {
         cmd: './gradlew',
-        args: ['test'],
+        args: ['test', '--stacktrace'],
```
Comment (victoralfaro-dotcms) [inline .github/actions/run-unit-tests/dist/index.js:130]: Same as comment above.

File: .github/actions/build-core/src/core-builder.ts (hunk 1 above)
Comment (wezell) [reply .github/actions/build-core/src/core-builder.ts:24]: If there are no errors, there won't be stack traces generated.

File: .github/actions/run-integration-tests/src/integration.ts (hunk 3 above)
Comment (wezell) [reply .github/actions/run-integration-tests/src/integration.ts:87]: see above

File: .github/actions/build-core/src/core-builder.ts (hunk 1 above)
Comment (spbolton) [reply .github/actions/build-core/src/core-builder.ts:24]: @victoralfaro-dotcms I am not sure if there is any reason not to add this parameter though.  I think it only impacts if we see the stack trace or not when there is an exception thrown by the core junit processing.   When there is no stacktrace shown we cannot diagnose and there is no further information other than re-running.  So adding stack trace all the time will not do anything when it runs correctly and will help diagnosis if it fails.

**Review by jdotcms — APPROVED** [review]

**Pull request merged**